samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/integration/helper/integrated/AssessmentGradeInfoProvider.java

```diff
@@ -110,15 +112,24 @@ private PublishedAssessmentIfc getPublishedAssessment(String id) {
         return a;
     }
 
-    public boolean isAssignmentDefined(String id) {
-        // SAM-3068 avoid looking up another tool's id
-        if (!StringUtils.isNumeric(id)) {
+    public boolean isAssignmentDefined(String externalAppName, String id) {
+    	    // SAM-3068 avoid looking up another tool's id
+    	    if (!StringUtils.isNumeric(id)) {
```

Comment:
this is deprecated use isCreatable()

Reply:
I don't see any deprecation in StringUtils:

https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isNumeric-java.lang.CharSequence-

Reply:
yeah my bad was thinking NumberUtils.isNumeric